samples/algorithms/repeat-until-success/RepeatUntilSuccess.csproj

```diff
@@ -0,0 +1,8 @@
+﻿<Project Sdk="Microsoft.Quantum.Sdk/0.11.2004.2825">
```

Comment:
If you update the version here to the latest release, `0.12.20070124` then you would get the newer behavior Chris referred to, such as the allowance to release qubits that have been measured but not explicitly reset.